src/lib/store.ts

```diff
@@ -3,24 +3,44 @@ import type { Form, Response, Team, Match, Field } from "$lib/types";
 import { persisted } from "svelte-local-storage-store";
 import { extractGroups } from "./serialize";
 
+//Details of which type of form is being used ???
 export const form = persisted<Form | null>("form", null);
+
 export const scout = persisted<string>("scout", "");
 export const teams = persisted<Record<number, Team>>("teams", {});
 export const matches = persisted<Record<number, Match>>("matches", {});
+
+//The currently active respose (the one the user sees on the form page)
 export const response = persisted<number | null>("response", null);
+
+//QR code stuff, worry about it later
 export const code = persisted<number | null>("code", null);
+
+//list of responses which are "active" eg. can be edited (not yet submitted)
 export const activeResponses = persisted<Record<number, Response>>(
   "activeResponses",
   {}
 );
+
+//something to do with grid, shouldn't be necessary this year
```

Comment:
Yeah so the grid won't be necessary but if there is a need for a form field that needs multiple data points then let me know because that is why the grid is much more complicated and needs special handling